requirements/README.rst

```diff
@@ -32,3 +39,38 @@ If you instead need to surgically *downgrade* a dependency:
 2. Run ``make compile-requirements``
 
 This is considerably safer than trying to manually edit the ``*.txt`` files, which can easily result in incompatible dependency versions.
+
+Inconsistent dependencies
+*************************
+
+You might be directed to this section if a consistent-dependencies PR check has failed.
```

Comment:
Note for reviewers: I feel mixed on having information specific to the check be discussed here in the requirements readme, but it was worse having it inline in the workflow yaml file. Ideally, this would end up somewhere more central and we can turn the check into a reusable workflow. Not sure what's best in the meantime.